.gitignore

```diff
@@ -4,5 +4,6 @@
 node_modules/
```

Comment:
please avoid adding random entries to the `.gitignore`

note the `scratch/` and `_scratch/` dirs, anything you put under those will be ignored